exercises/matrix/.meta/src/reference/java/Matrix.java

```diff
@@ -1,11 +1,11 @@
 import java.util.regex.Pattern;
 
-public class Matrix {
+ class Matrix {
     private int[][] matrix;
     private static Pattern spacePattern = Pattern.compile(" ");
     private static Pattern newlinePattern = Pattern.compile("\\n");
 
-    public Matrix(String matrixAsString) {
+     public Matrix(String matrixAsString) {
```

Comment:
This can also be package private :)

Reply:
@FridaTveit thank you for suggestions! I will take care of it.